testlist.py

```diff
@@ -0,0 +1,88 @@
+from unittest import makeSuite, TestSuite
+
+
+
+class ipv8_testsuite(TestSuite):
```

Comment:
You should be able to use this: https://docs.python.org/2/library/unittest.html#unittest.TestLoader.loadTestsFromModule so you don't have to specify all of these manually.

Reply:
Then you could also just get rid of this entire file.